### ISSUE TITLE
Image Viewer doesn't respect rounded corner preference

### ISSUE THREAD

**Issue by gokhanyildirimlar** [linuxdeepin/deepin-image-viewer#58]
### Bug description
Image Viewer doesn't respect rounded corner preference and keeps the default corner radius all the time.

**Expected behavior:** Corner radius should change according to preference as in other applications.

### Screenshot
![Screen Capture_select-area_20201128160135](https://user-images.githubusercontent.com/45146702/100516191-3e4b2e00-3193-11eb-8897-3344dcd493f1.png)

**Package version:** v5.6.3.69-1

**Comment by deepin-CD** [comment]
This issue is being verified

**Comment by deepin-CD** [comment]
The new version does not have this problem, please try after 5.6.3.73

**Comment by gokhanyildirimlar** [comment]
> The new version does not have this problem, please try after 5.6.3.73

Thanks. For now, the issue remains with v5.6.3.73-1. I am going to try once I receive an update for deepin-image-viewer and close the issue if it works as it should.

**Comment by deepin-CD** [comment]
This issue will be temporarily closed due to no response received. Please reopen the issue if there're any new updates.

**Comment by gokhanyildirimlar** [comment]
> This issue will be temporarily closed due to no response received. Please reopen the issue if there're any new updates.

Well, the team didn't even roll an update to push a newer version to users. Also, I responded as you can see above. Anyway, it's fixed if you say so. Closing the issue is fine with me.

```
deepin-image-viewer:
  Installed: 5.6.3.73-1
  Candidate: 5.6.3.73-1
```

**Comment by gokhanyildirimlar** [comment]
Issue remains. Just installed Deepin v20.2 and I saw that this is not resolved.

**Comment by gokhanyildirimlar** [comment]
> Can you tell me how the file manager doesn't look like a fillet and where it is set? I haven't found a place to control the fillet property

I am gonna do better and show you. I apologize for the sound quilty. I didn't have time to fix microphone settings. 


https://user-images.githubusercontent.com/45146702/113248185-1ac0fd00-92c5-11eb-8d16-5f4ee61f800e.mp4

**Comment by dependon** [comment]
OK,Thank you

**Comment by deepin-CD** [comment]
Thanks for your feedback. 
We have recorded your problem and will fix it.

**Comment by dependon** [comment]
I fixed the bug at the end of last month.So it hasn't been integrated in version 5.7.8, so it should be right to wait until version 5.7.9.This bug was caused by a wrong submission three years ago. It's a bit hidden. Ha ha

**Comment by gokhanyildirimlar** [comment]
> I fixed the bug at the end of last month.So it hasn't been integrated in version 5.7.8, so it should be right to wait until version 5.7.9.This bug was caused by a wrong submission three years ago. It's a bit hidden. Ha ha

It's nice to hear. I am going to close the issue alongside the next update after confirmation. Good job, keep up the ironing DDE.  :)

**Comment by dependon** [comment]
> > 我已在上个月底修复了该错误，因此尚未将其集成到5.7.8版本中，因此应该等到5.7.9版才是正确的。此错误是三年前提交错误的原因引起的。它有点隐藏。哈哈
> 
> 很高兴听到。确认后，我将在下次更新的同时关闭该问题。干得好，保持熨烫DDE。:)
ha-ha

**Comment by gokhanyildirimlar** [comment]
The issue is fixed within the latest internal testing(28.5.2021). Thanks.
